.ci/scripts/validate.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
is this required?

Reply:
it is recommended, to avoid use a different shell and help IDEs to identified the syntax to use.

Reply:
IIUC, this particular script is not used in this repo. In case it's required, the pre-commit should be the one in charge and as it's used from the library itself, it won't be required to be created in any specific repo.

Reply:
I misunderstood what you asked about required, I was talking about the header `#!/usr/bin/env bash`, and you were talking about the script file.